### PR TITLE
Add get/set dial mode

### PR DESCRIPTION
**Pull request by WimDoze**
Add get/set dial mode api calls

### CODE REVIEW

File: Library/Services/Implementation/NfieldSurveysService.cs [hunk 1]
```diff
@@ -581,4 +608,9 @@ internal class UpdateSurvey
         public string InterviewerInstruction { get; set; }
     }
 
+    internal class UpdateDialMode
```
Comment (behnazbeaumont) [inline Library/Services/Implementation/NfieldSurveysService.cs:611]: perhaps DialModeModel would be a better name since it's used in both set and get?

Comment (WimDoze) [reply Library/Services/Implementation/NfieldSurveysService.cs]: Changed the name

**Comment by kefishev** [comment]
Don't forget to update version.txt. I would do it as part of this PR.

**Comment by WimDoze** [comment]
> Don't forget to update version.txt. I would do it as part of this PR.

I always do that when I merge to master, need to check then anyway.

**Review by kefishev — APPROVED** [review]

**Pull request merged**